lib/py/src/transport/TSocket.py

```diff
@@ -91,12 +91,13 @@ def isOpen(self):
             try:
                 peeked_bytes = self.handle.recv(1, socket.MSG_PEEK)
             except (socket.error, OSError) as exc:  # on modern python this is just BlockingIOError
-                if exc.errno == errno.EWOULDBLOCK:
+                if exc.errno in (errno.EWOULDBLOCK, errno.EAGAIN):
```

Comment:
Would this cause trouble if `errno.EWOULDBLOCK` and `errno.EAGAIN` is the same thing? I assume it won't but just to double check here :)

Reply:
Looks like this would be fine:
```
>>> foo = 1
>>> foo in (1, 1, 2, 3)
True
```